faq/prooffacehelp.php

```diff
@@ -679,16 +679,42 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 </p>
 
 
+<h3>Search/Replace Popup</h3>
+
+<p>
+    To make it easier to make repetative changes, this tool has been provided.
+    Click on the Search/Replace button and a small window pops up,
+    containing a search value and a replace value.
+</p>
+<p>
+    Clicking <input type='button' value='Replace all.'> will replace all
+    all matched instances of the search text with the value from the replace text.
+    You may undo this change by clicking <input type='button' value='Undo.'>, but only
+    the most recent replace operation may be reverted.
```

Comment:
I realize that the instructions here are just matching the buttons, but, can we lose the periods terminating the buttons?